Microsoft.Azure.CosmosRepository.Samples/Paging/Paging.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>net6.0</TargetFramework>
+        <LangVersion>latest</LangVersion>
+        <UserSecretsId>9c9f04ba-0288-4b94-a15e-c669da85dfad</UserSecretsId>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Bogus" Version="33.0.2" />
+      <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="5.0.2" />
```

Comment:
If you're TFM is `net6.0`, your `PackageReference` could probably target that too.

Reply:
Happy to bump it :)